torch/fx/passes/pass_manager.py

```diff
@@ -1,188 +0,0 @@
-from functools import wraps
-from inspect import unwrap
-from typing import Callable, List
-
-
-# for callables which modify object inplace and return something other than
-# the object on which they act
-def inplace_wrapper(fn: Callable) -> Callable:
```

Comment:
this is a BC breaking change. 
fbcode has dependency on this function https://www.internalfb.com/code/fbsource/[d3381e07aff67fc001520c7e2018816b27cc8993]/fbcode/deeplearning/trt/torch_tensorrt/py/torch_tensorrt/fx/passes/lower_pass_manager_builder.py?lines=6%2C98%2C110

Let's talk offline. we will need a special workflow for merging this PR.

Reply:
same for other changes that removed code.